testsuite/features/secondary/min_centos_salt.feature

```diff
@@ -80,23 +90,23 @@ Feature: Be able to bootstrap a CentOS minion and do some basic operations on it
     And I wait until event "OpenSCAP xccdf scanning" is completed
 
 @centos_minion
-  Scenario: Run a remote command on the CentOS minion
+  Scenario: Run a remote command on the SSH-managed CentOS minion
     Given I am authorized as "testing" with password "testing"
     When I follow the left menu "Salt > Remote Commands"
     Then I should see a "Remote Commands" text in the content area
     When I enter command "cat /etc/os-release"
     And I enter target "*centos*"
```

Comment:
Out of scope: Not sure about it, but if that target is looking for the name of the VM, we might want to refactor it, so we pass `"centos_ssh_minion" hostname` as a target. Just thinking in the future if we have someday more VMs matching "*centos*" in this feature's test.